iRacingSDK.Net/CrossThreadEvents.cs

```diff
@@ -0,0 +1,144 @@
+﻿// This file is part of iRacingSDK.
+//
+// Copyright 2014 Dean Netherton
+// https://github.com/vipoo/iRacingSDK.Net
+//
+// iRacingSDK is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// iRacingSDK is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with iRacingSDK.  If not, see <http://www.gnu.org/licenses/>.
+
+
+using System;
```

Comment:
I am curious, why can't we just reference .dll built and keep the source code outside of CC code?   Or, is it because of GPL?

Reply:
Cause i need to modifty the source as its a bit outdated.